src/Controller/SubscriptionController.php

```diff
@@ -36,17 +36,31 @@ class SubscriptionController extends ControllerBase {
    */
   protected $messenger;
 
+  /**
+   * Entity type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface|mixed|object
+   *    Gets storage of user.
+   */
+  protected $entityTypeManager;
+
   /**
    * Constructs a SubscriptionController object.
    *
    * @param \Drupal\og\OgAccessInterface $og_access
    *   The OG access service.
    * @param \Drupal\Core\Messenger\MessengerInterface $messenger
    *   The messenger service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager service.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    */
-  public function __construct(OgAccessInterface $og_access, MessengerInterface $messenger) {
+  public function __construct(OgAccessInterface $og_access, MessengerInterface $messenger, EntityTypeManagerInterface $entity_type_manager) {
```

Comment:
In the `SubscriptionController` we don't need to inject the `EntityTypeManager` since we already inherit the `$this->getEntityTypeManager()` method from the base class (`ControllerBase`).